rakefile.netstd.rb

```diff
@@ -106,9 +106,17 @@
 end
 
 desc "Restore NuGet packages"
-exec :restore do |cmd|
-  cmd.command = nuget_command
-  cmd.parameters "restore #{solution} -PackagesDirectory #{packages}"
+task :restore do
+  restore_cmd = Exec.new
+  restore_cmd.command = nuget_command
+  restore_cmd.parameters "restore #{solution} -PackagesDirectory #{packages}"
+  restore_cmd.execute
+
+  # performing restore on the solution doesn't restore

```

Comment:
What exactly do you find bizarre? The fact that we have to specify the solution *and* the packages directory?